assets/js/modules/analytics-4/datastore/properties.js

```diff
@@ -801,36 +824,30 @@ const baseSelectors = {
 	},
 
 	/**
-	 * Checks if properties are currently being loaded.
+	 * Checks if properties summaries are currently being loaded.
 	 *
 	 * This selector was introduced as a convenience for reusing the same loading logic across multiple
 	 * components, initially the `PropertySelect` and `SettingsEnhancedMeasurementSwitch` components.
 	 *
-	 * @since 1.111.0
+	 * @since n.e.x.t
 	 *
 	 * @param {Object}  state                Data store's state.
 	 * @param {Object}  args                 Arguments object.
 	 * @param {boolean} args.hasModuleAccess Whether the current user has access to the Analytics module(s).
 	 */
-	isLoadingProperties: createRegistrySelector(
+	isLoadingPropertySummaries: createRegistrySelector(
 		( select ) =>
 			( state, { hasModuleAccess } ) => {
-				const accountID = select( MODULES_ANALYTICS ).getAccountID();
-
-				const isResolvingProperties =
-					hasModuleAccess === false || ! accountID
+				const isResolvingAccountSummaries =
```

Comment:
I think we can return early here if `hasModuleAccess` is `true` and we hasn't finished resolution for account summaries. In this case, we can avoid calling additional selectors.

Reply:
Updated ✅